### DISCUSSION TITLE
Add new command create to easily ceeate an empty helmfile

### DISCUSSION THREAD

**Discussion by pgrunm** [helmfile/helmfile#836] [Ideas]
Hello, 

I'd like to suggest a new command like `helmfile create`. This would enable users to create an empty Helmfile without loosing time for copying or searching again for the template.

I'm thinking of something simple like the Taskfile project does:

```bash
task -i
# https://taskfile.dev

version: '3'

vars:
  GREETING: Hello, World!

tasks:
  default:
    cmds:
      - echo "{{.GREETING}}"
    silent: true
 created in the current directory
```

**Comment by yxxhero** [comment]
@xiaomudk I remember that you post a similar idea before.

**Comment by jouve** [comment]
I'd suggest user snippet.

example for [vscode](https://code.visualstudio.com/docs/editor/userdefinedsnippets)
